packages/vscode-extension/src/chat/utils.ts

```diff
@@ -50,7 +50,11 @@ export function countMessageTokens(message: LanguageModelChatMessage): number {
     if (!value) {
       continue;
     }
-    numTokens += tokenizer.tokenLength(value);
+    if (key === "role") {
```

Comment:
There is  no need to count token size of `role`.  The `BaseTokensPerMessage` just stands for that. 
> By the way: if wanna count the size of `role`, you should use the real role value like "system", "user" and "assistant"

Reply:
Just notice the ChatParticipant SDK has exposed the [interface](https://github.com/microsoft/vscode/blob/5f78b58b57b7cf84d28d801fed6bb4a48f908601/src/vscode-dts/vscode.proposed.languageModels.d.ts#L166) for counting token size. 

Maybe we could adopt to that later.

Reply:
Thanks. I've removed the token counting for the `role`. 

`computeTokenLength` will be supported later in another PR.